From b6c74ece0b0b754a7db2c6ace960fd63dfb83b48 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 11 Apr 2017 10:05:35 +0800
Subject: [PATCH 1662/5242] MLK-14655-5 video: mxsfb: remove ofb var clear
 when open ofb

commit  031e5b2cad002ad0b2bfd49b317c6602509c5bd5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Clear the variable screeninfo structure when the overlay
fb is opened has the side effect of deleting timings info
of overlay fb.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 9fd7656d75de224d3e825930cd3f894408fbad40)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 9f8ae3d..91a2020 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1700,8 +1700,6 @@ static int overlayfb_open(struct fb_info *info, int user)
 	struct mxsfb_info  *fbi = ofb->fbi;
 
 	if (atomic_inc_return(&ofb->usage) == 1) {
-		memset((void*)&info->var, 0x0, sizeof(info->var));
-
 		ofb->ol_fb->var.xres		= fbi->fb_info->var.xres;
 		ofb->ol_fb->var.yres		= fbi->fb_info->var.yres;
 		ofb->ol_fb->var.xres_virtual	= fbi->fb_info->var.xres_virtual;
-- 
1.7.9.5

